scripts/aspell-pws

```diff
@@ -177,3 +177,4 @@ kfifo
 PTR
 ttt
 tictactoe
+vmalloc
```

Comment:
Be aware of the newline character.
See https://dev.to/documendous/why-you-should-end-your-source-files-with-a-new-line-156g